filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -294,6 +294,17 @@ void OpenGLDriver::setRasterStateSlow(RasterState rs) noexcept {
         gl.depthMask(GLboolean(rs.depthWrite));
     }
 
+    // stencil test / operation
+    if (rs.stencilFunc == RasterState::StencilFunction::A && !rs.stencilWrite) {
+        gl.disable(GL_STENCIL_TEST);
+    } else {
+        gl.enable(GL_STENCIL_TEST);
+        gl.stencilFunc(getStencilFunc(rs.stencilFunc), rs.stencilRef, ~GLuint(0));
```

Comment:
why is the mask set to all 1's here?

Reply:
If I understand correctly, this mask is separate from `glStencilMask`. I suppose we could add it to the API as well, but the default is all 1's. According to the spec:

> Specifies a mask that is ANDed with both the reference value and the stored stencil value when the test is done. The initial value is all 1's.

Reply:
maybe we just make it part of the API? Maybe it could be used so that we could reserve some bits for us vs. some bits for the users?

Reply:
Yeah that sounds good.

Reply:
Do you have any thoughts on where to store the mask value? Even with 64 bits, `RasterState` doesn't have enough to store a full 32 bit mask, let alone two masks (a stencil read mask and write mask).

Reply:
After discussing offline the resolution is to ignore the mask values for now.